### PR TITLE
[main] Fix selectors from submission tests

### PR DESCRIPTION
**Pull request by chamons**
These selectors failed the most recent submission test:
```
UIFocusDebugger
  checkFocusGroupTreeForEnvironment:

HMMatterRequestHandler
  configureAccessoryWithName:room:completion:
  fetchRoomsInHome:completion:
  pairAccessoryInHome:onboardingPayload:completion:
HMMatterTopology
  initWithHomes:
HMMatterHome/HMMatterRoom
  initWithUUID:name:
HMAccessorySetupManager
  performMatterEcosystemAccessorySetupUsingRequest:topology:completionHandler:

ChipDeviceController
  pairDevice:address:port:discriminator:setupPINCode:error:
  setListenPort:
  sharedController
ChipLowPower
  sleep:
ChipTestCluster
  test:
  toggle:
ChipGeneralCommissioning
  commissioningComplete:
```
Passed API diff locally:

image

### CODE REVIEW

**Review by dalexsoto — APPROVED** [review]
Submission tests are happy now!

**Comment by chamons** [comment]
Let's try it one more time....

/sudo backport xcode14.1

**Review by mandel-macaque — APPROVED** [review]

**Comment by mandel-macaque** [comment]
crazy 🤖

**Comment by mandel-macaque** [comment]
🤖 🤖

File: src/Chip/ChipCompat.cs [hunk 1]
```diff
@@ -138,5 +139,351 @@ public virtual Task<ChipReadAttributeResult> GoToTiltValueAsync (ushort tiltValu
 #endif
 
 	}
+
+	[Mac (12,1), Watch (8,3), TV (15,2), iOS (15,2), MacCatalyst (15,2)]
+	public delegate void ChipDeviceConnectionCallback (ChipDevice device, NSError error);
+
+	[Register("CHIPDeviceController", true)]
```
Comment (rolfbjarne) [inline src/Chip/ChipCompat.cs:146]: No big deal, but maybe we'll want `[Obsolete]` attributes on these classes as well (like we have for other classes in this file)?

File: src/HomeKit/HMCompat.cs [hunk 2]
```diff
@@ -26,84 +30,99 @@ namespace HomeKit {
...
+		public virtual void AddAndSetUpAccessories (HMChipServiceTopology topology, HMErrorHandler completion) => throw new InvalidOperationException (Constants.RemovedFromHomeKit);
+		public virtual Task AddAndSetUpAccessoriesAsync (HMChipServiceTopology topology) => throw new InvalidOperationException (Constants.RemovedFromHomeKit);
 #pragma warning restore CS0618 // HMChipServiceTopology and HMErrorHandler is obsolete
+	}	
+#endif // !NET
 
-	} /* class HMAccessorySetupManager */
-
+#if __IOS__ && !__MACCATALYST__
+	public unsafe partial class HMAccessorySetupManager : NSObject {
```
Comment (rolfbjarne) [inline src/HomeKit/HMCompat.cs:113]: Same: add Obsolete?

**Review by rolfbjarne — APPROVED** [review]

File: src/HomeKit/HMCompat.cs (hunk 2 above)
Comment (chamons) [reply src/HomeKit/HMCompat.cs:115]: Good idea - done.

File: src/Chip/ChipCompat.cs (hunk 1 above)
Comment (chamons) [reply src/Chip/ChipCompat.cs:147]: Good idea - done.

**Comment by chamons** [comment]
Test fixed (and XAMCORE_5_0 added as discussed in the call).

**Pull request merged**